ansible/roles/debops.prosody/defaults/main.yml

```diff
@@ -0,0 +1,162 @@
+---
+
+
+prosody__deploy_state: "present"
+
+
+# Packages and installation [[[
+# -----------------------------
+
+# .. envvar:: apache__base_packages [[[
+#
+# List of base packages to install.
+prosody__base_packages:
+  - prosody
+  - lua-zlib
+  - prosody-modules # for ldap auth
+#  - lua-event # for use_libevent = true
+                                                                   # ]]]
+                                                                   # ]]]
+
+
+# PKI [[[
+# ~~~~~~~
+
+# .. envvar:: prosody_pki [[[
+#
+# Enable or disable support for PKI/SSL/TLS in prosody.
+# Defaults to ``True`` if debops.pki_ is enabled on the remote host.
+prosody_pki: '{{ ansible_local|d() and ansible_local.pki|d() and
```

Comment:
Ensure that all role variables are properly namespaced with a `prosody__` prefix.